### ISSUE TITLE
Rate settings fields in node,shifts to the right as you enter values

### ISSUE THREAD

**Issue by Roberrtt** [BeamMW/beam#1190]
Make the field fixed .
Make it possible to enter only once the digit 0 if it is the first.
[Video2.zip](https://github.com/BeamMW/beam/files/3976928/Video2.zip)
![Screenshot_340](https://user-images.githubusercontent.com/50007056/71061587-63f54800-2179-11ea-85dd-dc6dfc0865ce.png)

**Comment by Roberrtt** [comment]
Make that be the fields(Username , Password) don't expand when you enter a lot of characters 
![Screenshot_491](https://user-images.githubusercontent.com/50007056/84005391-bf103680-a975-11ea-841f-defa6a5542c0.png)

**Comment by chapati** [comment]
Both seems to be fixed in new settings.
Though new issue is reproduced now: fee rate input is moved when rate is less than 5 characters
